Makefile

```
@@ -6,7 +6,17 @@ install: ## Install the Poetry environment
 .PHONY: test
 test: ## Run unit tests
 	@echo "Running unit tests"
-	@poetry run pytest tests
+	@poetry run pytest tests --cov=chispa --cov-report term 
```

```suggestion
	@poetry run pytest tests --cov=chispa --cov-report=term 
```

Because in other places we are using it with `=`